posthog/queries/test/test_trends.py

```diff
@@ -542,12 +542,38 @@ def test_trends_compare(self):
             self.assertEqual(response[0]["data"][4], 3.0)
             self.assertEqual(response[0]["labels"][5], "day 5")
             self.assertEqual(response[0]["data"][5], 1.0)
+            self.assertEqual(
+                response[0]["days"],
+                [
+                    "2019-12-28",
+                    "2019-12-29",
+                    "2019-12-30",
+                    "2019-12-31",
+                    "2020-01-01",
+                    "2020-01-02",
+                    "2020-01-03",
+                    "2020-01-04",
+                ],
+            )
 
+            self.assertEqual(
+                response[1]["days"],
+                [
+                    "2019-12-21",
```

Comment:
We are changing the behaviour of compare here, but 2020-01-04 minus 14 days is 21 so this makes a lot more sense